testsuite/dynaconf_loader.py

```diff
@@ -109,6 +123,17 @@ def load(obj, env=None, silent=None, key=None):
             server_url=ocp_setup.get("server_url"),
             token=ocp_setup.get("token"))
 
+        apicast_ocp_setup = \
+            weakget(obj)["threescale"]["gateway"]["OperatorApicast"]["openshift"]["servers"]["default"] % ocp_setup
```

Comment:
this is nice, just wanted to ask @pehala to check this, is this correct "path" to potential server definition (I think we dont need/use it now). I would probably say "no" as it looks to deep to me.